ecl/plugins/org.eclipse.rcptt.ecl.core/src/org/eclipse/rcptt/ecl/internal/commands/IfService.java

```diff
@@ -38,8 +41,14 @@ public IStatus service(Command command, IProcess context)
 		if (branch == null) {
 			return Status.OK_STATUS; // nothing to do
 		}
-		return session.execute(branch, context.getInput(), context.getOutput())
+		return session.execute(wrapBody(branch), context.getInput(), context.getOutput())
 				.waitFor();
 	}
 
+	private static Command wrapBody(Command body) {
```

Comment:
What is the purpose of this?
Do we want to give an opportunity to use this variable to the user? Or we only want to make the report more clearly?
I only see that if this variable will be declared, we will be able to distinguish 'then' and 'else' branches from each other in the report.